<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPS_MainWindow</class>
 <widget class="QMainWindow" name="LPS_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LightPlan Studio 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>231</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="lightPlanTab">
     <attribute name="title">
      <string>LightPlan Explorer</string>
     </attribute>
     <widget class="QTreeView" name="lp_tree_view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sslTab">
     <attribute name="title">
      <string>SSL Queue</string>
     </attribute>
     <widget class="QTableWidget" name="ssl_queue_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LightPlanBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>LightPlan</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Song Title:  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_songtitle_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_artist_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Song Artist:  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="lp_notes_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LP Author:  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_author_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SSL ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_sslid_edit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_spotifyid_edit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Spotify ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Event Table:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Youtube URL:  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_youtubeurl_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="lp_save_button">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LightPlan Start:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="start_event_edit">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00.000</string>
     </property>
    </widget>
    <widget class="QTableView" name="event_table_view">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>521</width>
       <height>321</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="streamDelayBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Stream Delay</string>
    </property>
    <widget class="QLCDNumber" name="delay_lcd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="calc_delay_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Runtime Adjust:</string>
     </property>
    </widget>
    <widget class="QSlider" name="delay_adjust_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-30000</number>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="delay_adjust_spinner">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="set_delay_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>sec</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="control_startlp_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start LightPlan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="twitch_connect_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Twitch</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="control_lp_progressbar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>Light Event: 0/0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elapsed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="control_lpelapsed_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Event:</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="control_nextevent_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="debug_log_edit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>431</width>
      <height>691</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debug_exportlog_button">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>730</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debug_hidelog_button">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>730</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Hide</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>541</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Event Wizard</string>
    </property>
    <widget class="QPushButton" name="insert_event_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert Event</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="current_time_lcd">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>9</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>123456789.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="load_song_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="play_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QStyle.SP_MediaPlay">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QStyle.SP_MediaPlay">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QSlider" name="song_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLabel" name="song_loaded_status_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>No Song Loaded</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="song_loaded_status_icon">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_new_lp"/>
    <addaction name="action_open_lp"/>
    <addaction name="action_save_lp"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_help_about"/>
    <addaction name="action_help_oauth"/>
    <addaction name="action_help_docs"/>
    <addaction name="action_help_checkcmds"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_show_log"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_help_about">
   <property name="text">
    <string>&amp;About LightPlan Studio</string>
   </property>
  </action>
  <action name="action_help_oauth">
   <property name="text">
    <string>&amp;Oauth Token Help</string>
   </property>
  </action>
  <action name="action_new_lp">
   <property name="text">
    <string>&amp;New LightPlan</string>
   </property>
  </action>
  <action name="action_open_lp">
   <property name="text">
    <string>&amp;Open LightPlan</string>
   </property>
  </action>
  <action name="action_Wizard">
   <property name="text">
    <string>&amp;LightPlan Creation Wizard</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_help_docs">
   <property name="text">
    <string>&amp;Online Help</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_lp_wizard">
   <property name="text">
    <string>LightPlan Wizard</string>
   </property>
  </action>
  <action name="action_show_log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Log</string>
   </property>
  </action>
  <action name="action_save_lp">
   <property name="text">
    <string>Save LightPlan</string>
   </property>
  </action>
  <action name="action_help_checkcmds">
   <property name="text">
    <string>&amp;Check for Updated Light Commands</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
